app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-finder-tracker.js

```diff
@@ -0,0 +1,137 @@
+//= require govuk/vendor/polyfills/Element/prototype/closest.js
+;(function (global) {
+  'use strict'
+
+  var GOVUK = global.GOVUK || {}
+  GOVUK.analyticsGa4 = GOVUK.analyticsGa4 || {}
+
+  GOVUK.analyticsGa4.Ga4FinderTracker = {
+
+    setFilterIndexes: function () {
+      var filterContainer = document.querySelector('[data-ga4-filter-container]')
+
+      if (!filterContainer) {
+        return
+      }
+
+      var filterSections = filterContainer.querySelectorAll('[data-ga4-section]')
+
+      for (var i = 0; i < filterSections.length; i++) {
+        var section = filterSections[i]
+        section.setAttribute('data-ga4-index', JSON.stringify({ index_section: i + 1, index_section_count: filterSections.length }))
+      }
+    },
+
+    trackChangeEvent: function (eventTarget, changeEventMetadata) {
+      var PIIRemover = new window.GOVUK.analyticsGa4.PIIRemover()
+
+      changeEventMetadata = changeEventMetadata.split(' ')
```

Comment:
Did you consider using more than one attribute to save some processing here? Or using a JSON string?

Reply:
My thinking was it would be less bytes added to the HTML if they were in one string, otherwise you'd have to add another `data-ga4-something` to each input in the HTML. Should I split it out?